code/modules/food&drinks/drinks/drinks/bottle.dm

```diff
@@ -389,3 +389,33 @@
 		user << "<span class='info'>You snuff out the flame on \the [src].</span>"
 		overlays -= fire_overlay
 		active = 0
+
+////////////////////////// POTIONS ///////////////////////
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/potion
+	name = "elixir flask"
+	desc = "My potions are too strong for you, traveler."
```

Comment:
Niknak is wrong, references are allowed, or rather are up to the maintainers discretion.

The actual ban is on player references and is not that strongly enforced.